spec/features/views_game_description_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+
+  feature 'user views the details of a game' do
+    let(:game) { FactoryGirl.create(:game) }
+    let(:game1) { FactoryGirl.create(:game1) }
+
+    scenario 'user views game details' do
+      sign_up
+      game
+      visit "/games"
+      click_link "#{game.name}"
```

Comment:
You don't need to use string interpolation here since `game.name` is the entire string:

```ruby
click_link game.name
```